round4/team-14.md

```diff
@@ -0,0 +1,105 @@
+# Chrome Extension Idea: Tamatimer
```

Comment:
I think this is a great idea, and also feasible at a basic level, but can definitely quickly scale in difficulty with your choice of customization options for the pets. I tried using the pomodoro method once, and I definitely cheated a couple times, so I could see myself using this to keep myself focused.

I'm interested in seeing how you will classify if a user is being productive. Some things I suggest to consider are the content of the tab they currently are and perhaps the amount of time their mouse is not moving.

As for the pet, I think the graphics will be the main challenge of this extension. I would recommend starting with simple static images for the pets. Once youre implementation of the "food" works, you consider some more advanced features. I would imagine users would like to interact with their pet, so consider looking into allowing the pet to follow their mouse or react to clicks. Alternatively, maybe consider group study sessions, where users can see all the pets of their friends in a group study session. Again, I think the complexity of this extension can scale quickly, so I would recommend deciding early on on what exact features you would like for the pets and figuring out how feasible they are to complete in 4 weeks.